remake.yml

```diff
@@ -34,21 +33,19 @@ targets:
     command: file.copy(from = "example_data/example_cars.csv", 
       to = target_name)
   
-
   out_data/spatial.zip:
     command: sf_to_zip(zip_filename = target_name, 
       sf_object = sf_spatial_data, layer_name = I('spatial_data'))
 
-  
-  sb_data:
+  out_xml/fgdc_metadata.xml:
+    command: render(filename = target_name,
+      "in_text/text_data_release.yml",
+      spatial_metadata)
+    
+  sb_posted:
     command: sb_replace_files(sbid,
       "out_data/spatial.zip",
       "out_data/cars.csv",
+      "out_xml/fgdc_metadata.xml",
```

Comment:
Also added this here, because with task tables, there is no longer a need to separate the xml metadata upload from the data upload. I was keeping those separate before because I didn't want to trigger large data uploads when I was messing around with metadata edits, but now the task table method only uploads files that have changed.